db/migrate/20180707065151_change_status_of_comments.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeStatusOfComments < ActiveRecord::Migration[5.2]
+  def up
+    Comment.all.each do |comment|
+      comment.status = 1
```

Comment:
what's this related to? Can't understand the context.

Reply:
Sometime before we made a migration for setting default column value as 1 for comment, in that PR statement for changing previous comment status are also included but that doesn't work. So, right now in production DB, there is a mix of comments with status =1 and 0. For the implementation of Spam moderation system, it's required that all comments should carry status 1, otherwise many comments wouldn't become invisible. @siaw23 do you think this migration would work?